app/controllers/api/v1/stats_controller.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V1
     class StatsController < ApiController
       def index
-        user = User.find params[:user_id]
+        user = User.where("id = ? OR rfid = ?", params[:user_id], params[:user_rfid]).first
```

Comment:
To maintain the same functionality this needs to use `find_by!` to trigger a RecordNotFound error